vault/dendron.topic.publish.config.md

```diff
@@ -1,63 +1,132 @@
 ---
 id: fabYbPyk3DMCMoG92lIrq
 title: Config
-desc: ''
-updated: 1640727025047
+desc: 'Configuration namespace that holds all Dendron Publishing related settings.'
+updated: 1645182877105
 created: 1637772506823
+config:
+  global:
+    enableChildLinks: false
 ---
 
 ## Summary
-
-Publishing configuration for Dendron Next.js Publishing.
-
-- NOTE: this is a work in progress, currently, refer to configuration options [[here|dendron://dendron.dendron-site/dendron.topic.publish-legacy.configuration]]
-
-## Scopes
-
-Configuration introduced here have three different scopes:
-- vault: applies over a vault
-- hierarchy: apply over a hierarchy
-- note: apply over a single note
-
-## Organization
-
-Controls the structure of notes in published page
-
-### Collection
-- scopes: [note]
-
-This feature is inspired by [jekyll collections](https://jekyllrb.com/docs/collections/).
-When set on a note, marks the given note and all children of it as a collection. 
-
-
-## SEO
-
-Search engine optimization related settings
-
-### Note Properties
-
-#### image
-- default: ""
-- syntax: 
-    - `image.url`: string
-    - `image.alt`: string
-- example
-    ```yml
+{{fm.desc}}
+
+Below is an overview of what the publishing namespace looks like:
+
+```yml
+publishing:
+  siteUrl:
+  siteRootDir:
+  siteIndex:
+  siteHierarchies:
+  hierarchy:
+  enableFMTitle:
+  enablePrettyRefs:
+  enableSiteLastModified:
+  enableNoteTitleForLink:
+  enableHierarchyDisplay:
+  enableKatex:
+  enableMermaid:
+  enableFrontmatterTags:
+  enableHashesForFMTags:
+  enableRandomlyColoredTags:
+  copyAssets:
+  assetsPrefix:
+  logoPath:
+  siteFaviconPath:
+  writeStubs:
+  customHeaderPath:
+  hierarchyDisplayTitle:
+  duplicateNoteBehavior:
+  ga:
+    tracking:
+  seo:
+    title:
+    description:
+    author:
+    twitter:
     image:
-        url: https://org-dendron-public-assets.s3.amazonaws.com/images/blog-mobile-editor-header.png
-        alt: A tree with many branches in full bloom
-    ```
-Sets image url for the page. Image shows up in image preview on social media sites
-
-
-## Static Assets
+      url:
+      alt:
+  github:
+    cname:
+    enableEditLink:
+    editLinkText:
+    editBranch:
+    editViewMode:
+    editRepository:
+```
+
+## Site properties
+These are properties related to how and where the site is being published, as well as what part of your workspace is being published.
+
+- [[siteUrl|dendron.topic.publish.config.siteUrl]]
```

Comment:
its not great that users need an extra click before they can access the config. as a followup CROP task, we should be able to materialize the text of the `desc` field of the target note inline if its configured on the note

Reply:
That's a good idea. I'll create a team crop for that

Reply:
`[[Materialize Frontmatter Description|dendron://private/task.2022.02.22.materialize-frontmatter-description]]`